samples/documentation/lulesh_sample1_unix_commented.yaml

```diff
@@ -0,0 +1,234 @@
+####################################
+# Description Block (Required)
+####################################
+# The description block is where the description of the study is placed. This
+# section is meant primarily for documentation purposes so that when a
+# spcecifcation is passed to other users they can gleam a general understanding
+# of what this study is meant to achieve.
+#-------------------------------
+# Required keys:
+#   name - Name of the study
+#   description - Description of what this study does.
+#-------------------------------
+# NOTE: You can add other keys to this block for custom documentation. Maestro
+# currently only looks for the required set.
+####################################
+description:
+    name: lulesh_sample1
+    description: A sample LULESH study that downloads, builds, and runs a parameter study of varying problem sizes and iterations.
+
+####################################
+# Environment Block
+####################################
+# The environment block is where items describing the study's environment are
+# defined. This includes static information that the study needs to know about
+# and dependencies that the workflow requires for execution.
+#-------------------------------
+# NOTE: This block isn't strictly required as a study may not depend on anything.
+####################################
+env:
+    #-------------------------------
+    # Variables
+    #-------------------------------
+    # Static values that the workflow substitutes into steps and are similar in
+    # concept to Unix environment variables. These variables are not dependent
+    # on values in the environment and so are more portable.
+
+    # NOTE: These values are substituted as static strings meaning that other
+    # variables should not used in other variables.
+    #-------------------------------
+    variables:
+        # OUTPUT_PATH is a keyword variable that Maestro looks for in order to
+        # set a custom output path for the study workspace. If not specified,
+        # OUTPUT_PATH is assumed to be the path where Maestro was launched from.
+        OUTPUT_PATH: ./sample_output/lulesh

```

Comment:
I have found this one a bit confusing; it's really more of a workspace_root, rather than an output_path. 
Renaming this variable would probably be best, but the description should certainly say that this is where ALL of maestro's temporary files will go, in addition to any output, and typically within a custom-named, date-stamped directory for this run of maestro.

Reply:
I corrected this in my last commit -- let me know if it's still unclear.